tests/Nelmio/Alice/Instances/Populator/PopulatorTest.php

```diff
@@ -22,7 +23,7 @@
 class PopulatorTest extends \PHPUnit_Framework_TestCase
 {
     const CONTACT = 'Nelmio\Alice\support\models\Contact';
-    const PLURAL = 'Nelmio\Alice\support\models\PluralProperties';
+    const PROPERTIES = 'Nelmio\Alice\support\models\Properties';
```

Comment:
No test uses the parent class directly, in both cases the child class is needed.

I just used the existing naming scheme for the method, will rename as you wish.